### ISSUE TITLE
NVDA not detecting objects with comments in PowerPoint for Office 365 as comments communicated via AnnotationObject UIA property

### ISSUE THREAD

**Issue by SMTestGH** [nvaccess/nvda#11662]
### Steps to reproduce:
Open PowerPoint desktop application
Select "blank presentation"
Tab to an object on the default slide
Press shift+f10 to open a context menu
Select "New  comment"
Type some filler text and press ctrl+enter to submit comment
Press F6 until focus lands back on "slide 1"
Tab again to the object with a comment

### Actual behavior:
NVDA does not announce that the object has a comment

### Expected behavior:
NVDA should listen for the AnnotationObject UIA property to report that an object has a comment. This is the new attribute that PowerPoint is leveraging to communicate that objects have comments on them.

### System configuration
Windows 10 64 bit
Core i7 4790
16gb RAM
Dell Precision T1700 workstation

#### NVDA installed/portable/running from source:
Installed

#### NVDA version: 
2020.2

#### Windows version:
Version 2004, build 19640.0 (issue is independent of Windows

#### Name and version of other software in use when reproducing the issue:
PowerPoint for Office 365 (reproduces on builds newer than version 2008, may require Office Insider)
#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Also reproduces in 2020.1

#### If addons are disabled, is your problem still occuring?
Yes

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No, not relevant in this case

**Comment by cary-rowen** [comment]
Yes, this problem does exist. Beta3 of VDA 2020.3 is now available for download and testing. If anyone is interested in trying out the next version of nvda before it is officially released, we welcome yo...

**Comment by SMTestGH** [comment]
@cary-rowen just wanted to confirm that this does still reproduce in NVDA 2020.3. Thanks!

**Comment by cary-rowen** [comment]
I still seem to be able to reproduce
